Runtime/Tracking/Follow/Modifier/Property/Rotation/RotateAroundAngularVelocity.cs

```diff
@@ -99,6 +106,12 @@ protected override void DoModify(GameObject source, GameObject target, GameObjec
             }
 
             Vector3 input = AngularVelocitySource.GetAngularVelocity();
+            
+            if (InTargetSpace) 
```

Comment:
Can combine this into a single line for neatness:

```
Vector3 input = InTargetSpace
    ? target.transform.parent.InverseTransformVector(AngularVelocitySource.GetAngularVelocity())
    : AngularVelocitySource.GetAngularVelocity();
```

Reply:
done.